x-pack/legacy/plugins/ingest/server/libs/policy.ts

```diff
@@ -258,7 +263,36 @@ export class PolicyLib {
       shared_id: newPolicyInfo.shared_id,
     };
   }
-
+  public async ensureDefaultPolicy() {
+    try {
+      await this.adapter.get(DEFAULT_POLICY_ID);
+    } catch (err) {
+      if (!err.isBoom || err.output.statusCode !== 404) {
+        throw err;
+      }
+      const info = this.libs.framework.info;
+      if (info === null) {
+        throw new Error('Could not get version information about Kibana from xpack');
+      }
+      const newDefaultPolicy: NewPolicyFile = {
+        name: 'Default Policy',
+        description: 'default policy create by kibana (not possible to delete)',
```

Comment:
looks like we plan on surfacing description in policy list UI, so I think we should make this description as polished as possible. in the policy list UI, we can disable the checkbox and delete action buttons and surface a tooltip or callout to explain that it's not possible to delete the default policy, so this description doesn't need to explain that it cannot be deleted.

with that in mind, I suggest changing this description to simply: `Default policy created by Kibana`

would localization be something we want to keep in mind for this as well? for both `name` and `description`

Reply:
100% agree, and yes great idea... we really should localize this for users as it is not user entered